projects/igniteui-angular/src/lib/toast/toast.component.ts

```diff
@@ -249,7 +262,7 @@ export class IgxToastComponent implements IToggleView, OnInit, OnDestroy {
      * @memberof IgxToastComponent
      */
     public get element() {
-        return this.elementRef.nativeElement;
+        return this.element.nativeElement;
```

Comment:
this should throw stack overflow.
This should either
 - get the element from `this.elementRef.nativeElement` and make `elementRef` protected in toggle.
 - get the element from `this._element.nativeElement` and `_element` private field in the constructor.